Specs/DateTime/Korean/DateParser.json

```diff
@@ -1919,38 +1879,14 @@
             "date": "1989-05-05"
           }
         },
-        "Start": 13,
-        "Length": 8
-      }
-    ]
-  },
-  {
-    "Input": "2071년 3월 7일에 돌아갈 거야. ",
```

Comment:
Wasn't the case with 2017 fixed added back?

Reply:
yes, but then was removed because duplicate